exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/internal/OtlpMetricExporterProvider.java

```diff
@@ -72,6 +81,7 @@ public MetricExporter createExporter(ConfigProperties config) {
           config, builder::setAggregationTemporalitySelector);
       ExporterBuilderUtil.configureOtlpHistogramDefaultAggregation(
           config, builder::setDefaultAggregationSelector);
+      builder.setMeterProvider(meterProviderRef::get);
```

Comment:
I feel like it would be good to make sure we have tests that cover this particular line. The test addition doesn't seem to verify that we're actually doing this very important step.